prisma/schema.prisma

```diff
@@ -0,0 +1,30 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:./dev.db"
```

Comment:
yup! we can add any provider we want, postgresql may be better as we could use specify membershiptype as an enum rather than a string as it is currently